packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -724,13 +740,16 @@ export class EcsOptimizedImage implements ec2.IMachineImage {
     }
 
     // set the SSM parameter name
-    this.amiParameterName = '/aws/service/ecs/optimized-ami/'
+    if (this.windowsVersion) {
+      this.amiParameterName = `/aws/service/ami-windows-latest/${this.windowsVersion}-ECS_Optimized/image_id`;
```

Comment:
This is a breaking change for users who already have an ssm parameter name in the format `/aws/service/ecs/optimized-ami/windows_server/<VERSION>/english/full` in their templates. So let's only use this format if the user supplies one of the new enum values defined in this PR.